lib/helpers.js

```diff
@@ -0,0 +1,21 @@
+import ms from 'ms'
+
+import { DEFAULT_OPTIONS } from './constants.js'
+
+export const convertToMs = time => {
+  if (typeof time === 'number') return time
+  if (typeof time === 'string') {
+    return ms(time)
+  }
+  throw new Error(
+    `The provided value: ${time} is unable to convert in milliseconds`
+  )
+}
+
+export const sleep = ms => new Promise(res => setTimeout(res, ms))
+
+export const slowDownAPI = async endpointReached => {
+  for (let i = 0; i < DEFAULT_OPTIONS.delayAfter; i++) {
+    await endpointReached()
+  }
+}
```

Comment:
this is a test helper, let's move it within a module inside the test folder as it's not production code